packages/core/BaseFeatureWidget/BaseFeatureDetail.tsx

```diff
@@ -66,7 +66,7 @@ export const useStyles = makeStyles()(theme => ({
     padding: theme.spacing(1),
   },
   expandIcon: {
-    color: '#FFFFFF',
+    color: theme.palette.tertiary?.contrastText || '#fff',
```

Comment:
Have you seen cases where `tertiary` is undefined?

Reply:
that was a little shortcut to bypass to avoid having to wrap some unit tests with ThemeProvider